cmake/simdutf-flags.cmake

```diff
@@ -8,8 +8,8 @@ if (NOT CMAKE_BUILD_TYPE)
   if(SIMDUTF_SANITIZE OR SIMDUTF_SANITIZE_UNDEFINED)
     set(CMAKE_BUILD_TYPE Debug CACHE STRING "Choose the type of build." FORCE)
     # SIMDUTF_SANITIZE only applies to gcc/clang:
-    message(STATUS "Setting debug optimization flag to -O1.")
-    set(CMAKE_CXX_FLAGS_DEBUG "-O1" CACHE STRING "" FORCE)
+    message(STATUS "Setting debug optimization flag to -O1 -g.")
+    set(CMAKE_CXX_FLAGS_DEBUG "-O1 -g" CACHE STRING "" FORCE)
```

Comment:
fwiw I usually do -g3 in debug builds. `-g` with optimizations still misses stuff